api-put-object.go

```diff
@@ -119,7 +122,9 @@ func (opts PutObjectOptions) Header() (header http.Header) {
 	if opts.ReplicationStatus != "" {
 		header.Set(amzBucketReplicationStatus, opts.ReplicationStatus)
 	}
-
+	if opts.ReplicationMTime != timeSentinel {
```

Comment:
can it not be `ReplicationMTime.IsZero()` ? @poornas more simpler

Reply:
definitely.